docs/topics/tour/kotlin-tour-collections.md

```diff
@@ -0,0 +1,361 @@
+        <img src="icon-3.svg" width="20" alt="Third step" /> <strong>Collections</strong><br />
+        <img src="icon-4-todo.svg" width="20" alt="Fourth step" /> <a href="kotlin-tour-control-flow.md">Control flow</a><br />
+        <img src="icon-5-todo.svg" width="20" alt="Fifth step" /> <a href="kotlin-tour-functions.md">Functions</a><br />
+        <img src="icon-6-todo.svg" width="20" alt="Sixth step" /> <a href="kotlin-tour-classes-part-1.md">Classes</a><br />
+        <img src="icon-7-todo.svg" width="20" alt="Final step" /> <a href="kotlin-tour-null-safety.md">Null safety</a></p>
+</microformat>
+
+When programming, it is useful to be able to group data into structures for later processing. Kotlin provides collections
+for exactly this purpose.
+
+Kotlin has the following collections for grouping items:
+
+| **Collection type** | **Description**                                                         |
+|---------------------|-------------------------------------------------------------------------|
+| Lists               | Ordered collections of items                                            |
+| Sets                | Unique collections of items                                             |
+| Maps                | Sets of key-value pairs where keys are unique and map to only one value |
+
+Each collection type can be mutable or read only.
+
+## List
+
+To create a read-only list ([`List`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-list/)), use the 
+`[listOf()](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/list-of.html)` function.
+
+To create a mutable list ([`MutableList`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/-mutable-list.html)),
+use the [`mutableListOf()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin.collections/mutable-list-of.html) function.
+
+When creating lists, Kotlin can sometimes infer the type of items stored. To declare the type explicitly, add the type
+within angled brackets after the list declaration.
+
+```kotlin
+fun main() { 
+//sampleStart
+    val readOnlyNumbers = listOf(1, 2, 3)                  //Read only list
+    val numbers: MutableList<Int> = mutableListOf(1, 2, 3) //Mutable list with explicit type declaration
+//sampleEnd
+}
+```
+{kotlin-runnable="true" kotlin-min-compiler-version="1.3" id="lists-declaration-kotlin"}
+
+> To prevent unwanted modifications, obtain read-only views of mutable lists by casting them to `List`.
+> ```kotlin
+>     val numbers: MutableList<Int> = mutableListOf(1, 2, 3)
+>     val numbersLocked: List<Int> = numbers
+> ```
+> 
+{type="tip"}
+
+To access an item in a list, use the [indexed access operator](operator-overloading.md#indexed-access-operator) `[]`:
+
+```kotlin
+fun main() { 
+//sampleStart
+    val readOnlyNumbers = listOf(1, 2, 3)
+    println("The first item in the list is: ${readOnlyNumbers[0]}")
```

Comment:
I believe this is the first time when we use `${}` as a template. Maybe, this example requires additional note. Alternatively, we could mention that it's allowed to write any expression inside templates in the previous chapter.